science/ngspice/Portfile

```diff
@@ -74,19 +80,6 @@ if {${name} eq ${subport}} {
         depends_run-append       port:ngspice-lib
     }
 
-    variant openmp description {Add OpenMP support variant} {
```

Comment:
Wouldn't removing this mean a much slower execution time because of the lack of parallelism?

Reply:
It does.  ngpspice has virtually no value as-is due to the fact that Apple doesn't support openmp out of the box.  I attempted to use an openmp enabled compiler, but I found the results to be incorrect.  ngspice-43 allows compatibility with skywater and other IC design kits.  I've been using ngspice-43 for advanced nodes.

Reply:
We could split the ports.

Reply:
I thought we have openmp enabled compilers?